src/RawTransaction.php

```diff
@@ -1188,6 +1189,7 @@ public static function extract_input_signatures_p2sh($input, $message_hash, $scr
                     foreach ($script_info['public_keys'] as $key) {
                         if (self::_check_sig($sig, $message_hash, $key) == true) {
                             $signatures[$key] = $sig;
+                            break;
```

Comment:
I think this may cause some breakages, as the number of sigs it extracts is factored into the sign_count value returned by sign(), and may be relied on to see if all signatures have been provided.